includes/documents/class-wcpdf-invoice.php

```diff
@@ -506,6 +506,38 @@ public function get_pdf_settings_fields( $option_name ) {
 					'description'	=> sprintf(__( "Disable document when the order total is %s", 'woocommerce-pdf-invoices-packing-slips' ), function_exists('wc_price') ? wc_price( 0 ) : 0 ),
 				)
 			),
+			array(
+				'type'			=> 'setting',
+				'id'			=> 'due_date',
+				'title'			=> __( 'Due date', 'woocommerce-pdf-invoices-packing-slips' ),
+				'callback'		=> 'select',
+				'section'		=> $this->type,
+				'args'			=> array(
+					'option_name'	=> $option_name,
+					'id'			=> 'due_date',
+					'options'       => apply_filters( 'wpo_wcpdf_due_date_options', array(
+						// For predefined dates, use this format: "{number_of_days}_day(s)".
+						'0_day'   => __( 'None', 'wpo_wcpdf_pro' ),
```

Comment:
Maybe this slugs could be:

- 1
- 7
- 30
- custom

instead? I think we could avoid having a `none` option there doing the same we do for the document date for example: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/88942c4a2667a355ec31e9f1956e72b24ab19227/includes/documents/class-wcpdf-invoice.php#L278